lib/core.js

```diff
@@ -23,6 +23,16 @@ const loadOptionsInConfigFileIfExists = async (path, options = {}, componentName
 	return options
 }
 
+const loadReacliConfigurationFileIfNotIgnored = async (ignoreConfigFile, path, options, componentName = "") => {
+	if (ignoreConfigFile) {
+		console.log(chalk.black.bgBlue("Ignore the optional Reacli configuration file."))
+	} else {
+		options = await loadOptionsInConfigFileIfExists(path, options, componentName)
```

Comment:
Same here... In my opinion, `loadOptionsInConfigFile` is greatly enough ☺️

Reply:
https://github.com/reacli/cli/pull/48/commits/6834c0917e02506b0492d5f7a2bead3c168de52f